internal/adapters/scheduler/scheduler_storage_pg.go

```diff
@@ -100,11 +101,14 @@ func (s schedulerStorage) GetScheduler(ctx context.Context, name string) (*entit
 	var dbScheduler Scheduler
 
 	queryString := queryGetActiveScheduler
+	start := time.Now()
```

Comment:
@arthur29 What's your opinion on having a wrapper function to be used in all of these methods for reporting latency metrics instead of calling it directly, something like:
```
func queryReportingLatency(storageOperation string, queryFunction func() error) error {
	start := time.Now()
	functionResult := queryFunction()
	reportSchedulerStorageLatency(start, storageOperation)
	return functionResult
}
```

And them when executing some query:
```
	err := queryReportingLatency("GetScheduler", func() error {
		_, err := client.QueryOne(&dbScheduler, queryString, name)
		return err
	})
```

Reply:
Also, we could even handle fail metrics in this wrapper: 
```
func queryReportingLatency(storageOperation string, queryFunction func() error) error {
	start := time.Now()
	err := queryFunction()
         if err != nil {
            reportSchedulerStorageFailsCounterMetric(storageOperation)
         }

	reportSchedulerStorageLatency(start, storageOperation)
	return err
}
```

Reply:
Awesome!